### PR TITLE
mosquitto: initial integration

### PR DESCRIPTION
**Pull request by ralight**
There is currently a different PR open (#8793), but the mosquitto project would prefer this implementation.

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
Why is mosquitto a critical project? Why should we accept it into OSS-Fuzz?

**Comment by ralight** [comment]
Mosquitto is probably the most widely used open source MQTT broker - it has [580M pulls on dockerhub](https://hub.docker.com/v2/search/repositories/?query=mosquitto&is_official=true) for example.

It is integrated into a huge variety of projects and products. For example, it is used in some [UK rail signalling infrastructure](https://www.youtube.com/watch?v=OTbpUbTO5_I), the [UK informatics system for mapping COVID-19 variants](https://twitter.com/MosquittoMQTT/status/1595134666700632064), is built into industrial controllers from a number of global manufacturers, and is integrated into the [Philips Hue bridge](https://community.home-assistant.io/t/native-mqtt-push-updates-from-hue-hub/299504). There are many more examples that I am not aware of. 

Other open source projects such as [HomeAssistant](https://www.home-assistant.io/integrations/mqtt/) use mosquitto as a key component of their project. The Mosquitto journal paper published with the Journal of Open Source Software has been [cited 468 times](https://scholar.google.co.uk/scholar?hl=en&as_sdt=0%2C5&q=%22mosquitto%22&oq=).

Will that do? :)

**Comment by jonathanmetzman** [comment]
> Mosquitto is probably the most widely used open source MQTT broker - it has [580M pulls on dockerhub](https://hub.docker.com/v2/search/repositories/?query=mosquitto&is_official=true) for example.
> 
> It is integrated into a huge variety of projects and products. For example, it is used in some [UK rail signalling infrastructure](https://www.youtube.com/watch?v=OTbpUbTO5_I), the [UK informatics system for mapping COVID-19 variants](https://twitter.com/MosquittoMQTT/status/1595134666700632064), is built into industrial controllers from a number of global manufacturers, and is integrated into the [Philips Hue bridge](https://community.home-assistant.io/t/native-mqtt-push-updates-from-hue-hub/299504). There are many more examples that I am not aware of.
> 
> Other open source projects such as [HomeAssistant](https://www.home-assistant.io/integrations/mqtt/) use mosquitto as a key component of their project. The Mosquitto journal paper published with the Journal of Open Source Software has been [cited 468 times](https://scholar.google.co.uk/scholar?hl=en&as_sdt=0%2C5&q=%22mosquitto%22&oq=).
> 
> Will that do? :)

That sounds very important. I'm asking the committee if we approve of adding this.

File: projects/mosquitto/project.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+language: c
+primary_contact: "roger@atchoo.org"
+vendor_ccs:
+ - "security@eclipse.org"
+auto_ccs:
+ - "rogerlight@gmail.com"
+sanitizers:
+ - address
+ - memory:
+    experimental: True
```
Comment (jonathanmetzman) [inline projects/mosquitto/project.yaml:10]: Did you copy this from somewhere? Otherwise I wouldn't mark msan as experimental.

**Review by jonathanmetzman — APPROVED** [review]
lgtm

File: projects/mosquitto/project.yaml (hunk 1 above)
Comment (ralight) [reply projects/mosquitto/project.yaml:10]: Yes, from the documentation:

https://google.github.io/oss-fuzz/getting-started/new-project-guide/#sanitizers

**Pull request merged**